plugins/slick.customtooltip.js

```diff
@@ -302,7 +326,14 @@
         // create the tooltip DOM element with the text returned by the Formatter
         _tooltipElm = document.createElement('div');
         _tooltipElm.className = _cellTooltipOptions.className + ' ' + _grid.getUID();
-        _tooltipElm.innerHTML = tooltipText || parseFormatter(formatter, cell, value, columnDef, item);
+        _tooltipElm.classList.add('l' + cell.cell);
+        _tooltipElm.classList.add('r' + cell.cell);
+        var outputText = tooltipText || parseFormatter(formatter, cell, value, columnDef, item);
+        if (_cellTooltipOptions.enableRenderHtml || !tooltipText || _cellTooltipOptions.renderRegularTooltipAsHtml) {
+          _tooltipElm.innerHTML = sanitizeHtmlString(outputText.replace(/[\r\n]/gi, '<br>'));
+        } else {
+          _tooltipElm.textContent = outputText;
```

Comment:
on the if condition, remove the replace of `\r\n` to `<br>` and instead use `_tooltipElm.style.whiteSpace = 'pre';` in the else condition